lib/utils/ConfigParser.js

```diff
@@ -272,6 +258,42 @@ class ConfigParser {
         return this._capabilities
     }
 
+    /**
+     * sets config attribute with file paths from filtering
+     * options from cli argument
+     *
+     * @param  {String} filterObj  list of files in a string from
+     * cli argument
+     * @return {Boolean} exclude  true or false
+     */
+    setFilePathToFilterOptions (filterObj, config) {
+        const objs = new Set()
+        const objList = filterObj.split(/,/g)
+        const objsList = ConfigParser.getFilePaths(config)
+
+        for (let obj of objList) {
+            if (fs.existsSync(obj) && fs.lstatSync(obj).isFile()) {
+                objs.add(path.resolve(process.cwd(), obj))
+                continue
+            }
+
+            const paramFiles = objsList.find(file => file.match(obj))
+            if (paramFiles) {
+                objs.add(paramFiles)
+            }
+        }
+
+        if (objs.size === 0) {
+            throw new Error(`File ${filterObj} not found`)
+        }
+
+        return objs
+    }
+
+    isTypeOfString (obj) {
```

Comment:
this method is so simple that I don't think we need to make a class function out of it

Reply:
yeah, a bit overhead... removed.